app/assets/javascripts/components/campaign/campaign_stats_download_modal.jsx

```diff
@@ -11,6 +11,20 @@ const CampaignStatsDownloadModal = ({ match }) => {
   const editorsLink = `/campaigns/${campaignSlug}/students.csv`;
   const editorsByCourseLink = `/campaigns/${campaignSlug}/students.csv?course=true`;
   const instructorsLink = `/campaigns/${campaignSlug}/instructors.csv?course=true`;
+  const wikidataLink = `/campaigns/${campaignSlug}/wikidata.csv`;
+
+  let wikidataCsvLink;
+  if (Features.wikiEd) {
```

Comment:
There's no need to limit this based on `Features.wikiEd`.

Reply:
I did that, because there is this limit in the course version (`app/assets/javascripts/components/overview/course_stats_download_modal.jsx`
So I implied it would be the same for a campaign. I will remove it.

Reply:
Ah! We should change that limitation on CourseStatsDownloadModal... now it should check for whether the `course` object has a `course_stats` property. @1v4n4's project made the wikidata stats data available without the Wiki Ed restriction.